### DISCUSSION TITLE
kant/user-guide/route/create-route

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2753] [Announcements]
# kant/user-guide/route/create-route

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kant/user-guide/route/create-route.html

**Comment by liyuerich** [comment]
点击删除，即删除成功，放回到消息路由列表页。

应该是“返回”到消息路由列表页？

**Comment by windsonsea** [comment]
Fixed in PR #5269
